sdmetrics/visualization.py

```diff
@@ -34,9 +34,127 @@ def _generate_column_bar_plot(real_data, synthetic_data, plot_kwargs={}):
         'pattern_shape_sequence': ['', '/'],
         'histnorm': 'probability density',
     }
+    histogram_kwargs.update(plot_kwargs)
     fig = px.histogram(
         all_data,
-        **{**default_histogram_kwargs, **plot_kwargs}
+        **histogram_kwargs
+    )
+
+    return fig
+
+
+def _generate_heatmap_plot(real_data, synthetic_data):
+    """Generate heatmap plot for discrete data.
+
+    Args:
+        real_data (pandas.DataFrame):
+            The real data for the desired column pair.
+        synthetic_column (pandas.Dataframe):
+            The synthetic data for the desired column pair.
+
+    Returns:
+        plotly.graph_objects._figure.Figure
+    """
+    columns = real_data.columns
+    real_data = real_data.copy()
+    real_data['Data'] = 'Real'
+    synthetic_data = synthetic_data.copy()
+    synthetic_data['Data'] = 'Synthetic'
+    all_data = pd.concat([real_data, synthetic_data], axis=0, ignore_index=True)
```

Comment:
This part seems to be the same for all the plots. Maybe it should be done in `get_column_pair_plot` instead?